src/xmipp/applications/scripts/tomogram_reconstruction/tomogram_reconstruction.py

```diff
@@ -130,68 +135,80 @@ def defineParams(self):
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method sirt --iter 20')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method sart')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method ossart')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method lsmr')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method asd-pocs')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method os-asd-pocs')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method awasd-pocs')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method irn-tv-cgls')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method ab-gmres')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method ab-gmres')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method hybrid_lsqr')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method lsqr')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method cgls')
         
 
     def readInputParams(self):
         '''
         In this function the parameters are read. For for information about their use see the help
         '''
-        
+
         self.fnTs          = self.getParam('--tiltseries')
         self.fnAngles      = self.getParam('--angles')
         self.thickness     = self.getIntParam('--thickness')
         self.useAstra      = self.checkParam('--astra')
-        self.method        = self.getParam('--method').lower()
+        self.method        = self.fixMethod(self.getParam('--method').lower())
         self.fnOut         = self.getParam('-o')
+        self.normalizeTi   = self.getParam('--normalize').lower()
         self.gpuId         = self.getIntParam('--gpu') if self.checkParam('--gpu') else 0
         
         if self.method == 'wbp':
```

Comment:
Is this necessary? Maybe you could just not offer wbp as an option to start with?